modules/stats.php

```diff
@@ -654,6 +654,11 @@ function stats_convert_image_urls( $html ) {
 	return $html;
 }
 
+function stats_convert_chart_urls_preg_replace_callback( $matches ) {
```

Comment:
Let's namespace this with `jetpack_` please. In order to make things easier, we can remove the `_replace_callback` in the end and instead specify that this is going to be used as a callback inside a docblock.

Reply:
Alright! I guess I tried to name the function consistently with the way all of the functions here are named without a `jetpack_` prefix here. Will update.

Reply:
Updated in adfad172ef07ff5b8b053a9b3d19b3bc2b2a03af